registrations/models.py

```diff
@@ -930,13 +930,14 @@ def create_web_store_payment(self):
 
         if checkout_url := api_response.get("checkoutUrl"):
             kwargs["checkout_url"] = get_checkout_url_with_lang_param(
-                checkout_url, getattr(contact_person, "service_language_id", "fi")
+                checkout_url,
+                getattr(contact_person, "service_language_id", None) or "fi",
```

Comment:
You could also use `settings.LANGUAGE_CODE ` instead of `fi`

Reply:
This is for the Talpa checkout UI where there are three supported languages ("fi", "sv" and "en") and "fi" is the default/fallback option there. So perhaps it is better to explicitly use "fi" here?